server/src/service/team.service.ts

```diff
@@ -79,8 +78,38 @@ export class TeamService extends LiteDataSource {
 				case PgErrorCode.UniqueViolation:
 					throw new ApolloError('This user already exists on this team')
 				default:
-					throw new ApolloError(`An error occurred when adding user: "${userId}" to team: "${teamId}"`)
+					throw new ApolloError(
+						`An error occurred when adding user: "${userId}" to team: "${teamId}"`,
+					)
 			}
 		}
 	}
+
+	async answerQuestion(teamId: string, answer: string, questionId: string) {
+		/* Get Team */
+		const team = await Team.getOne(teamId)
+
+		/* Get Question */
+		const question = await Question.getOne(questionId)
+
+		const isCorrect = question?.answers?.find(correctAnswer => correctAnswer.value === answer) ? true : false
```

Comment:
I think this can be simplified to:
```suggestion
		const isCorrect = !!question?.answers?.some(correctAnswer => correctAnswer.value.toLowerCase() === answer.toLowerCase()
```
I also lowercased both the correct answer and the submission